### DISCUSSION TITLE
i dont have 023000EE i have 02300024 why??

### DISCUSSION THREAD

**Discussion by djmarkino94** [Mr-MIBonk/M.I.B._More-Incredible-Bash#27] [Q&A] [answered]
hello I followed the procedure to the letter, I installed the mib on the green menu but when I check the fec I don't have the correct one 023000eefor unlimited maps but only the one there, 02300024.  what should i do to have the correct fec?  i have a rs3 8v 2019 thanks in advance

**Comment by harman-f** [comment]
Share your log file.

**Reply by djmarkino94** [reply]
could you explain me how to do it?  I'm sorry but it's all new

**Comment by harman-f** [comment]
In M.I.B SD 
/Backup/logs

**Reply by djmarkino94** [reply]
then I insert the sd with which I loaded the mib then I go to the backup log and I have to take a picture something?  or fsccio backup and then from the pc on the sd I find the log?

**Comment by harman-f** [comment]
Upload the .log file here

**Reply by djmarkino94** [reply]
Ok tomorrow i try it Thank's

**Reply by djmarkino94** [reply]
I go to the car now

**Reply by djmarkino94** [reply]
- [ ]  i Have posted it

**Comment by djmarkino94** [comment, marked as answer]
[logs.zip](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/9336623/logs.zip)
[MU1106-JTB-07405.10.1711820306.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/9336627/MU1106-JTB-07405.10.1711820306.log)

**Reply by djmarkino94** [reply]
let me you know thank you so much, the backup file im not able to upload, its too big around 115mb

**Reply by harman-f** [reply]
Just looked at the log file. So far you only created a backup. 
No patch, no coding.

Please run the AIO patch function.

**Reply by djmarkino94** [reply]
sorry but I did not understand, I also followed the video and it does not show other steps, where should I run this AIO patch?  could you be clearer?  I would be grateful

**Reply by djmarkino94** [reply]
> Just looked at the log file. So far you only created a backup. 
> 
> No patch, no coding.
> 
> 
> 
> Please run the AIO patch function.

sorry but I did not understand, I also followed the video and it does not show other steps, where should I run this AIO patch?  could you be clearer?  I would be grateful

**Reply by harman-f** [reply]
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/Prepare-M.I.B-SD-card-&-install-M.I.B#3-do-stuff-smirk

**Reply by djmarkino94** [reply]
so i just have to go to the green menu and press this command?(((Flash patched image | add FECs | CP/AA coding)))
does it all by itself then?  does it change the fec in automatic? can I install the map directly?

**Reply by harman-f** [reply]
FEC and patch, yes
Map is a seperate install

**Reply by djmarkino94** [reply]
I had the patch L I had already inserted in the sd when I prepared L, do I have to install another one?  and as for the fec?  do I have to select a specific one or just hit the command string mentioned in your guide in the green menu?

**Reply by harman-f** [reply]
Yes, it is really that simple. 
Not idea what you mena by "L"

SD card has of course be prepared correctly with patch files on SD

**Reply by djmarkino94** [reply]
yes of course I meant the sd, however I have already installed the mib via the red menu with the correct patch already inserted, I don't have to repeat the procedure right?  if it was not successful I would not see the item "MIB" appear on the green menu right?  so now it remains for me to press this button: (((Flash patched image | add FECs | CP / AA coding))) in the green menu and does it all by itself?  already install the correct patch and then install the maps with the sd that I have already prepared?  thanks sorry but I'm Italian and not so familiar with English.  thanks for your time

**Reply by harman-f** [reply]
Just go for it.

**Reply by djmarkino94** [reply]
yes but I would not like to block something understood?  I wanted to know if this fec fixes itself once I press the button in the MIB

**Comment by harman-f** [comment]
Yes

**Reply by djmarkino94** [reply]
Ok i try it and il let you know Thank's in advance

**Reply by djmarkino94** [reply]
I managed to put the version and and thank you very much today I try to install the maps

**Reply by djmarkino94** [reply]
All its complete Thank you so much
